tool/cluster-cli/main.go

```diff
@@ -0,0 +1,71 @@
+package main
```

Comment:
what about moving the `tool/cluster-cli` and `tool-cli` packages into a single `cmd` package, which is what is usually done with command line interfaces?  
Note that `spf13/cobra` already uses this naming: https://github.com/spf13/cobra#create-your-maingo -> see the `"{pathToYourApp}/cmd"` pkg in the example

Reply:
I believe, in every project, we are using the same convention. I don't want to break it. 

I can move it manually here. but again I couldn't find a way using which goa can automatically generate cli in one package using `--tool` and `--tooldir`

Reply:
ok, fair enough, then